web/client/epics/geostory.js

```diff
@@ -116,10 +116,7 @@ const updateMediaSection = (store, path) => action$ =>
             if (resourceAlreadyPresent) {
                 resourceId = resourceAlreadyPresent.id;
             } else if (isEmpty(resource)) {
-                const emptyMedia = mediaType === MediaTypes.MAP
-                ? {resourceId: '', type: null, map: undefined}
-                : {resourceId: '', type: null};
-                actions = [...actions, update(`${path}`, emptyMedia ), hide()];
+                actions = [...actions, remove(`${path}`), hide()];
```

Comment:
@jstrR It seems the `hide()` function is called twice, is the first one needed?

https://github.com/geosolutions-it/MapStore2/blob/f22b653d2dd8d162907840f625402d55c5e515e2/web/client/epics/geostory.js#L119

https://github.com/geosolutions-it/MapStore2/blob/f22b653d2dd8d162907840f625402d55c5e515e2/web/client/epics/geostory.js#L126

Reply:
@allyoucanmap yes, first hide() is needed